tests/reg_tests/test_adjoint.py

```diff
@@ -172,6 +172,69 @@ def span(val, geo):
         "aero_prob": ap_tutorial_wing,
         "evalFuncs": ["fx", "mz", "cl", "cd", "cmz", "lift", "drag"],
     },
+    # # Rotating frame test
+    {
+        "name": "Rotating_wing",
+        "options": {
```

Comment:
I think some of these options can be removed, like the ones related to multigrid and NK. It doesn't look like either of those solvers are used. If we ever need to recreate the restart file, this would make it easier to see which non-default options were used.